.tekton/integration-tests/integration-test-scenarios/compliance-operator-fbc-4-18-integration-tests-arm-mixarch-destuctive.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: appstudio.redhat.com/v1beta2
+kind: IntegrationTestScenario
+metadata:
+  annotations:
+    test.appstudio.openshift.io/optional: "true"
+  name: compliance-operator-fbc-v4-18-e2e-tests-arm-mixarch-destructive
```

Comment:
None of the other ITS were named with `mixarch` in the name.
I'm fine with or without `mixarch`, but we need to be consistent.